### DISCUSSION TITLE
How to disable error messages showing up on page?

### DISCUSSION THREAD

**Discussion by lethargistic** [pyscript/pyscript#1618] [Q&A] [answered]
I need to stop this from appearing each time I get an error.

![Screenshot_4](https://github.com/pyscript/pyscript/assets/58907570/21ee2b23-c050-4616-8268-dd388620c5ef)

**Comment by JeffersGlass** [comment, marked as answer]
One sledgehammer of an option would be to hide them using css:
```html
<style>
  .py-error{
    display: none;
  }
</style>
```

**Reply by lethargistic** [reply]
Should do it, can I also replace the loading screen in the same way?
